README.md

```diff
@@ -96,76 +148,54 @@ models:
 
-### Disabling Models
-
-This package takes into consideration that not every Zendesk account utilizes the `schedule`, `domain_name`, `user_tag`, `organization_tag`, or `ticket_form_history` features, and allows you to disable the corresponding functionality. By default, all variables' values are assumed to be `true`. Add variables for only the tables you want to disable:
+> IMPORTANT: See this project's [`dbt_project.yml`](https://github.com/fivetran/dbt_zendesk_source/blob/main/dbt_project.yml) variable declarations to see the expected names.
 
 ```yml
-# dbt_project.yml
-
-...
-config-version: 2
-
 vars:
-  using_schedules:            False         #Disable if you are not using schedules
-  using_domain_names:         False         #Disable if you are not using domain names
-  using_user_tags:            False         #Disable if you are not using user tags
-  using_ticket_form_history:  False         #Disable if you are not using ticket form history
-  using_organization_tags:    False         #Disable if you are not using organization tags
+    zendesk_<default_source_table_name>_identifier: your_table_name 
 ```
-*Note: This package only integrates the above variables. If you'd like to disable other models, please create an [issue](https://github.com/fivetran/dbt_zendesk/issues) specifying which ones.*
-
-### Extending and Limiting the Ticket Field History
-
-This package will create a row in `zendesk__ticket_field_history` for each day that a ticket is open, starting at its creation date. A Zendesk ticket cannot be altered after being closed, so its field values will not change after this date. However, you may want to extend a ticket's history past its closure date for easier reporting and visualizing. To do so, add the following configuration to your `dbt_project.yml` file:
-
+</details>
+
+## (Optional) Step 6: Orchestrate your models with Fivetran Transformations for dbt Core™
+<details><summary>Expand for details</summary>
+<br>
+    
+Fivetran offers the ability for you to orchestrate your dbt project through [Fivetran Transformations for dbt Core™](https://fivetran.com/docs/transformations/dbt). Learn how to set up your project for orchestration through Fivetran in our [Transformations for dbt Core setup guides](https://fivetran.com/docs/transformations/dbt#setupguide).
+</details>
+
+# 🔍 Does this package have dependencies?
+This dbt package is dependent on the following dbt packages. Please be aware that these dependencies are installed by default within this package. For more information on the following packages, refer to the [dbt hub](https://hub.getdbt.com/) site.
+> IMPORTANT: If you have any of these dependent packages in your own `packages.yml` file, we highly recommend that you remove them from your root `packages.yml` to avoid package version conflicts.
+    
 ```yml
-# dbt_project.yml
+packages:
```

Comment:
We should also add spark utils here now.

Reply:
added here and to the source README